src/org/zaproxy/zap/extension/httpsinfo/ExtensionHttpsInfo.java

```diff
@@ -80,8 +82,9 @@ public void unload() {
 		unloaded = true;
 
 		while (!dialogues.isEmpty()) {
-			// When disposed the dialogue is removed from the list
-			dialogues.get(0).dispose();
+			for (HttpsInfoDialog dialog : dialogues) {
```

Comment:
K got the loop loop part. 

How do I address the other part? Can I somehow prevent the user from launching further dialogs after uninstalling?